config/redis.yml

```diff
@@ -1,9 +1,9 @@
 development:
```

Comment:
Maybe out of scope for this. But this is a pretty useless file TBH. Probably was given to us by auto generating from sufia.

If we wrote our redis initializer smarter we wouldnt need this. The Redis Gem already configures localhost and port 6379 as the default. Also it supports the usage of the `ENV['REDIS_URL']` Env var which should be used instead here.